src/error-code.ts

```diff
@@ -686,6 +686,76 @@ export namespace ErrorCodes {
       type: ErrorType.ERROR,
       title: "Unable to find reference out of application function",
     };
+
+  /**
+   * Appsync Integration invocations must be deterministic.
```

Comment:
I think deterministic is the wrong word. Each of these cases are deterministic based on input. I think you mean they must always be statically defined? Or "resolvers cannot be dynamically called in loops"

I'd consider avoiding this word and instead just exhaustively list what isn't supported.